lib/ash/plug_helpers.ex

```diff
@@ -144,5 +175,36 @@ if Code.ensure_loaded?(Plug.Conn) do
 
     def get_tenant(%{private: %{ash: %{tenant: tenant}}}), do: tenant
     def get_tenant(_), do: nil
+
+    @doc """
+    Updates the tenant inside the Plug connection.
+
+    The tenant is stored inside the [connection's private
+    fields](https://hexdocs.pm/plug/Plug.Conn.html#module-private-fields).
+
+    ## Example
+
+        iex> tenant = build_tenant(%{name: "Deliver-yesterday"})
+        ...> conn = build_conn() |> put_private(:ash, %{tenant: tenant})
+        ...> tenant = get_tenant(conn)
+        %{name: "Deliver-yesterday"} = tenant
+        ...> conn = update_tenant(conn, fn tenant -> Map.put(tenant, :slogan, "You know we can.") end)
+        ...> tenant = get_tenant(conn)
+        %{name: "Deliver-yesterday", slogan: "You know we can."} = tenant
+        ...> conn = update_tenant(conn, fn tenant -> Map.delete(tenant, :name) end)
+        ...> tenant = get_tenant(conn)
+        %{slogan: "You know we can."} = tenant
+    """
+    @spec update_tenant(Conn.t(), (nil | Ash.Resource.record() -> nil | Ash.Resource.record())) :: Conn.t()
+    def update_tenant(conn, callback) do
```

Comment:
I don't think we need this. Tenants are always strings :) The other one looks good though.

Reply:
Tenants may not always only be strings, but for now they are.

Reply:
Well, setting tenant and getting the tenant matters, but if its just a string, there isn't really any "updating" to do. You'd just `set` it.